### ISSUE TITLE
[Feature]: ERC20Wrapper extension

### ISSUE THREAD

**Issue by bidzyyys** [OpenZeppelin/rust-contracts-stylus#354]
### What is the feature you would like to see?

Add ERC20Wrapper extension ([solidity](https://docs.openzeppelin.com/contracts/5.x/api/token/erc20#ERC20Wrapper)).

### Contribution Guidelines

- [X] I agree to follow this project's Contribution Guidelines

**Comment by Ifechukwudaniel** [comment]
@bidzyyys i think i can do this one , it looks like erc4626

**Comment by 0xNeshi** [comment]
@Ifechukwudaniel thanks for your interest!

Since your `ERC4626` PR #465 is still in progress and needs some review items addressed, let's get that one merged first before starting on a new issue. 

Once it's completed, we'll be happy to assign you either this issue, or the other one you've taken an interest in.
